browser/speedreader/speedreader_tab_helper.h

```diff
@@ -63,6 +63,7 @@ class SpeedreaderTabHelper
   // Returns |true| if the user has enabled Speedreader but the domain in the
   // active web contents is blacklisted.
   bool IsEnabledForSite();
+  bool IsEnabledForSite(const GURL& url);
```

Comment:
maybe we can make it private if nobody except the tab helper wants it

Reply:
fixed